### ISSUE TITLE
(AWS::Lambda::Function.FileSystemConfigs.LocalMountPath) `The local mount path must start with /mnt/ and be a valid absolute path.` not being checked

### ISSUE THREAD

**Issue by r-heimann** [aws-cloudformation/cfn-lint#2859]
### Is this feature request related to a new rule or cfn-lint capabilities?

New capability

### Describe the feature you'd like to request

The following CFN template doesn't check the path for `LocalMountPath`:

```yaml
LambdaFunction:
  Type: AWS::Lambda::Function
  Properties:
    FileSystemConfigs:
      - Arn: <arn>
        LocalMountPath: /efs
```

According to the Lambda GUI the path needs to start with `/mnt/`:

```
Local mount path
Only absolute paths are supported.
The local mount path must start with /mnt/ and be a valid absolute path.
```

### Describe the solution you'd like

cfn-lint should give you a notification if the path doesn't start with `/mnt/`:

`The local mount path must start with /mnt/ and be a valid absolute path.`

### Additional context

cfn-lint v0.79.7

### Is this something that you'd be interested in working on?

- [ ] 👋 I may be able to implement this feature request

### Would this feature include a breaking change?

- [ ] ⚠️ This feature might incur a breaking change

**Comment by r-heimann** [comment]
Here is the official documentation:

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-lambda-function-filesystemconfig.html

```
LocalMountPath
The path where the function can access the file system, starting with /mnt/.

Required: Yes

Type: String

Maximum: 160

Pattern: ^/mnt/[a-zA-Z0-9-_.]+$

Update requires: No interruption
```
